_opensearch/query-dsl/index.md

```diff
@@ -12,16 +12,60 @@ redirect_from:
 
 # Query DSL
 
-While you can use HTTP request parameters to perform simple searches, you can also use the OpenSearch query domain-specific language (DSL), which provides a wider range of search options. The query DSL uses the HTTP request body, so you can more easily customize your queries to get the exact results that you want.
+OpenSearch provides a query domain-specific language (DSL) that you can use to perform customized searches on your data. You can also combine query fields to perform more complex searches and find more granular data.
 
-For example, the following request performs a simple search to search for a `speaker` field that has a value of `queen`.
+OpenSearch supports the following query DSL query type categories:
+
+- **Compound queries** – Used to perform combined queries with the following query types:
+    - **Boolean** `bool` – Combines several simple query terms by Boolean clause rules that act as logical operators. To learn more, see [Boolean queries]({{site.url}}{{site.baseurl}}/opensearch/query-dsl/bool/).
+    - **Constant score** `constant_score` – Provides a `filter` query with the relevance score as a floating point number assigned by the `boost` field. Acts as a wrapper for the filter.
+    - **Disjunction max** `dis_max` – Returns documents that match one or more queries that are wrapped together by clauses. If a document matches multiple query clauses, it gets the highest relevant score assigned to it.
+    - **Function score** `function_score` – Modifies the score of documents that are returned by a query. You define a query and one or more functions to compute a score for each document that matches the query.
+
+- **Full-text queries** – Used to search documents for one or more terms and filter with advanced options. These queries perform text analysis and return detailed information, including relevance scores for each document match. To learn more, see [Full-text queries]({{site.url}}{{site.baseurl}}/opensearch/query-dsl/full-text/).
```

Comment:
If we're listing all the subtypes of the compound query, should we list all the subtypes of the full text query here?

Reply:
Because the compound query types listed here do not have topic pages we can link to (such as full-text), they need to be included in the index bullet list of descriptions.

The goal is to have a concise bullet list of category descriptions and link to their topic pages (if available).

Reply:
Other option is to create a sub-page for compound queries that would only contain these five type definitions.